fsharp-backend/tests/Tests/ApiServer.Tests.fs

```diff
@@ -988,4 +988,4 @@ let cookies =
        (302, None, Some "/login?error=Invalid+username+or+password"))
       (local, None, (302, None, Some "/login?error=Invalid+username+or+password")) ]
 
-let tests = testList "ApiServer" [ localOnlyTests; permissions; cookies ]
```

Comment:
Should we delete `localOnlyTests` while we're at it, or are you thinking to update those such that we run them generally? If so, let's make a separate Issue for that.

Reply:
Yeah, we should test that these work. Issue added